src/Models/Brand.php

```diff
@@ -37,7 +37,7 @@ protected static function booting()
                 })
                 ->leftJoinSub($renamedStore, 'store_value', function ($join) {
                     $join->on('store_value.sub_option_id', '=', 'eav_attribute_option.option_id')
-                         ->where('store_value.store_id', (Statamic::isCpRoute() ? Site::selected()->attributes['magento_store_id'] : Site::current()->attributes['magento_store_id']));
+                         ->where('store_value.store_id', (Statamic::isCpRoute() ? (Site::selected()->attributes['magento_store_id'] ?? '1') : (Site::current()->attributes['magento_store_id'] ?? '1')));
```

Comment:
Why the `?? 1` , maybe we also need a `validate` / healthcheck command that checks if the mandatory config values are set.

Reply:
You can check out https://github.com/rapidez/paynl/pull/21/files#diff-372b727c0d9abc808461b82d701bcd8df3609a9396e050bd5f21ad6515d05f05 to see how to make the health check.

You can add as many messages as you like and determine wether is means a health check fail when you want.

Reply:
Noted, will look into this later!